src/Log/ValueObjects/LogCategory.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace Give\Log\ValueObjects;
+
+/**
+ * Class LogCategory
+ * @package Give\Log\ValueObjects
+ *
+ * @since 2.9.7
+ */
+class LogCategory {
```

Comment:
Is this going to become a Value Object? Right now it's just a kind of namespaced constants.

Reply:
Yeah, I wanted to create a Value Object from these classes, but I give up. LogType Value Object might make sense, but I will simplify things for now and just move them into Log directory. 

Resolved a89ff89a